src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -478,6 +478,14 @@
   <data name="Arg_MustBeChar" xml:space="preserve">
     <value>Object must be of type Char.</value>
   </data>
+  <data name="Arg_MustBeDateOnly" xml:space="preserve">
+    <value>Object must be of type DateOnly.</value>
+    <comment>{Locked="DateOnly"}</comment>
+  </data>
+  <data name="Arg_MustBeTimeOnly" xml:space="preserve">
+    <value>Object must be of type TimeOnly.</value>
+    <comment>{Locked="TimeOnly"}</comment>
```

Comment:
Interesting... I don't think I've seen this in our resx files before.  Is this meant to convey to someone localizing that these terms shouldn't be localized?  Is this standardized somewhere?

Reply:
Yes, Microsoft localization tool can recognize such elements. I don't think this is standardized outside of that.

Reply:
If that's still how loc do things then maybe we should progressively add these as we make future changes. It's easy to do and means less work if we localize one day.